app/i18n/en/gen.php

```diff
@@ -14,7 +14,7 @@
 		'import' => 'Import',
 		'manage' => 'Manage',
 		'mark_read' => 'Mark as read',
-		'mark_favorite' => 'Mark as favourite',
+		'mark_favorite' => 'Mark as favorite',
```

Comment:
Erf, I've always thought it was "o" for British and "ou" for American. I got it all wrong all my life during... but anyway, thanks, I've learned something today :)

I think there are other parts where I put "o" instead of "ou". What shall we do? I close PR and re-make them?